aws/table_aws_kms_key.go

```diff
@@ -378,6 +379,23 @@ func getAwsKmsKeyAliases(ctx context.Context, d *plugin.QueryData, h *plugin.Hyd
 	return keyData, nil
 }
 
+func kmsKeyTitle(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("kmsKeyTitle")
+	key := d.HydrateItem.([]*kms.AliasListEntry)
+
+	var keyID string
+	if d.HydrateResults["listKmsKeys"] != nil {
+		keyID = *(d.HydrateResults["listKmsKeys"]).(*kms.KeyListEntry).KeyId
+	} else if d.HydrateResults["getKmsKey"] != nil {
+		keyID = *(d.HydrateResults["getKmsKey"]).(*kms.KeyListEntry).KeyId
+	}
+
+	if len(key) > 0 {
+		return key[0].AliasName, nil
+	}
+	return keyID, nil
+}
```

Comment:
```suggestion
func kmsKeyTitle(ctx context.Context, d *transform.TransformData) (interface{}, error) {
	key := d.HydrateItem.([]*kms.AliasListEntry)
	
	if len(key) > 0 {
		return key[0].AliasName, nil
	}

	var keyID string
	if d.HydrateResults["listKmsKeys"] != nil {
		keyID = *(d.HydrateResults["listKmsKeys"]).(*kms.KeyListEntry).KeyId
	} else if d.HydrateResults["getKmsKey"] != nil {
		keyID = *(d.HydrateResults["getKmsKey"]).(*kms.KeyListEntry).KeyId
	}

	return keyID, nil
}
```

A few suggestions:
- We probably don't need that Debug message (turns out it's mainly noisy in the logs when debugging and not very helpful).
- If we always prefer alias, we should do a check for that first and return it earlier if we have it.